nodecore-spv/src/main/java/org/veriblock/spv/SpvContext.kt

```diff
@@ -119,7 +120,8 @@ class SpvContext {
             network = config.network
         }
 
-        val peerDiscovery = if (config.useLocalNode) {
+        assert (networkParameters.name == config.network);
```

Comment:
Asserts in production code are no longer a thing (in jvm world at least). The way to solve this could be 
```suggestion
        check(networkParameters.name == config.network) {
              "SPV input Network Parameters are different from the configured network!"
        }
```